src/services/message.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Service, Container } from 'typedi';
+import { getRepository } from 'typeorm';
+import { Message } from '@entities/message.entity';
+import { ConversationsService } from './conversation.service';
+import { Conversation } from '@entities/conversation.entity';
+
+@Service()
+export class MessagesService {
+  private readonly messageRepository = getRepository<Message>(Message);
+
+  async sendMessage(message: Message) {
+    const conversationService = Container.get(ConversationsService);
+    const conversation = new Conversation();
+    conversation.user1Id = message.senderId;
+    conversation.user2Id = message.receiverId;
+    return await conversationService.existsConversation(conversation) ?
```

Comment:
I think that the readability here is a little tricky because a return with an away, I know that is because you are using the ternary to validate what you will return, but it is not easy to read. I think that you can improve the readability with something like
`const conversationExists = await conversationService.existsConversation(conversation);
const response = conversationExists ? this.messageRepository.save(message) : null;
return response
`
WDYT?